index.js

```diff
@@ -44,15 +44,6 @@ module.exports = {
     //   return failBuild(`This plugin cannot support apps that manually use next-on-netlify. Uninstall next-on-netlify as a dependency to resolve.`);
     // }
 
-    const isFunctionsDirectoryCorrect =
-      FUNCTIONS_SRC !== undefined && path.resolve(FUNCTIONS_SRC) === path.resolve('out_functions')
```

Comment:
`const path = require('path')` on top of the file can now be removed, I believe.